lib/msal-common/src/network/ThrottlingUtils.ts

```diff
@@ -54,10 +54,9 @@ export class ThrottlingUtils {
                 errorMessage: response.body.error_description,
                 subError: response.body.suberror
             };
-            cacheManager.setItem(
+            cacheManager.setThrottlingCache(
```

Comment:
Have `setThrottlingCache` generate the key

Reply:
same as above!